assignments/objects.js

```diff
@@ -18,27 +19,33 @@ const example = {
 }
 
 // Write your intern objects here:
-
+people =[{'id': 1, 'name': 'Mitzi', 'email':'mmelloy0@psu.edu', 'gender':'F'}, {'id': 2, 'name': 'Kennan', 'email':'kdiben1@tinypic.com', 'gender':'M'},
+{'id': 3, 'name': 'Keven', 'email':'kmummery2@wikimedia.org', 'gender':'M'}, {'id': 4, 'name': 'Gannie', 'email':'gmartinson3@illinois.edu', 'gender':'M'},
+{'id': 5, 'name': 'Antonietta', 'email':'adaine5@samsung.com', 'gender':'F'}];
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(people[0].name)
 // Kennan's ID
-
+console.log(people[1].id)
 // Keven's email
-
+console.log(people[2].email)
 // Gannie's name
-
+console.log(people[3].name)
 // Antonietta's Gender
-
+console.log(people[4].gender)
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log('Hello, my name is '+people[1].name);
```

Comment:
this is totally valid since it works, but the goal here was to add a function (method) to your kennan object. that would look like adding a key to kennan's object with your function name (like speak) but without quotes, and a value of a function that console.logs his hello message. then you could invoke it here with people[1].speak()